app/src/components/Fields/DateRangeField.js

```diff
@@ -45,11 +38,8 @@ export default function DateRangeField(props) {
 
   function renderSelectValue(value) {
     if (value && (value.to || value.from)) {
-      return `${
-        value.from ? extractTimeAndDateForDisplay(value.from).date : 'All dates'
-      } to ${
-        value.to ? extractTimeAndDateForDisplay(value.to).date : 'All dates'
-      }`;
+      return `${value.from ? formatISODate(value.from) : 'All dates'} to 
```

Comment:
Display of the date as DD-MM-YYYY  in this select box so that it matches the date boxes.

Reply:
Good point! Will do that

Reply:
@punkael Actually looks like there is a bug where the dates reset when the popover is dismissed

Reply:
I'll look into it